ci_scripts/get_config.py

```diff
@@ -0,0 +1,37 @@
+"""Script retrieving configuration values."""
+import argparse
+import sys
+import logging
+from utils.configuration import configuration, ConfigurationVariable
+from utils.logging import set_log_level, log_exception
+
+logger = logging.getLogger(__name__)
+
+
+def main():
```

Comment:
Happy to see argparsing happening in `main()`. It's one of the things we need to do, so that these files can be turned into executable scripts in the separate `-ci` repo, as `if __name__ == '__main__':` does not work there. 👍

Reply:
do I need to do anything here?

Reply:
Nope, I'm just expressing my happiness.

Reply:
Ah, I haven't been doing that. I will change it in the docs generation script when I add in the release sync stuff.